cfgmgr/intfmgr.cpp

```diff
@@ -518,6 +523,12 @@ bool IntfMgr::doIntfGeneralTask(const vector<string>& keys,
                 FieldValueTuple fvTuple("nat_zone", nat_zone);
                 data.push_back(fvTuple);
             }
+            /* Set mpls */
+            if (!mpls.empty())
+            {
+                FieldValueTuple fvTuple("mpls", mpls);
+                data.push_back(fvTuple);
```

Comment:
emplace_back will avoid un-necessary copy

Reply:
This is consistent with all other FieldValueTuple processing in this function.

Reply:
For code formatting style its ok to be consistent, need not be for code improvement.